### PR TITLE
feat: show configure button when metrics are not configured

### LINKED ISSUE #930: UI: Show 'Configure' action buttons when Grafana or Prometheus is not configured
**Current Behavior**

Grafana and Prometheus cards show "Not connected to \<Grafana\>" when not configured by the user.

**Desired Behavior**

It will be great if there is an action button which redirects the user to configure it instead.

---
**Resources**

**Alternatives / Additional Context**

Earlier discussed in #918

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #930

### CODE REVIEW

**Comment by kushthedude** [comment]
@leecalcote Screenshot for change : 
Screenshot 2020-05-02 at 8 40 03 AM

**Comment by Nikhil-Ladha** [comment]
> @leecalcote Screenshot for change :
> Screenshot 2020-05-02 at 8 40 03 AM

Everything looks good. Just a small suggestion from my side if that seems correct to you too that if you can add a bit of padding or gap between the settings icon and the configure text. It would just look a bit better.🙂

**Comment by kushthedude** [comment]
Seems appropriate, I was thinking the same thing.

**Comment by kushthedude** [comment]
@Nikhil-Ladha Have a look!

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha Have a look!
> Screenshot 2020-05-02 at 1 30 40 PM

Better! 👍🏼

File: ui/components/DashboardComponent.js [hunk 1]
```diff
@@ -209,6 +207,10 @@ class DashboardComponent extends React.Component {
     }, self.handleError('Could not ping adapter.'));
   }
 
+    handleConfigure = () => {
+    this.props.router.push('/settings#metrics');
```
Comment (leecalcote) [inline ui/components/DashboardComponent.js:211]: Nice! - - `'/settings#metrics`

Teamwork, @kushthedude @aayushdutt. 🧑‍🤝‍🧑

**Pull request merged**